plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -314,7 +314,7 @@ func (r *NodeAuthorizer) authorizeResourceSlice(nodeName string, attrs authorize
 	switch verb {
 	case "get", "create", "update", "patch", "delete":
 		// Okay, but check individual object permission below.
-	case "watch", "list":
+	case "watch", "list", "deletecollection":
```

Comment:
/assign ritazh

assigned as part of SIG Auth triage meeting to review node authorizer changes.